ui/StatusQ/src/wallet/managetokensmodel.h

```diff
@@ -20,10 +22,12 @@ const auto kEnabledNetworkBalanceRoleName = QByteArrayLiteral("enabledNetworkBal
 const auto kEnabledNetworkCurrencyBalanceRoleName = QByteArrayLiteral("enabledNetworkCurrencyBalance");
 const auto kCustomSortOrderNoRoleName = QByteArrayLiteral("customSortOrderNo");
 const auto kTokenImageRoleName = QByteArrayLiteral("imageUrl");
+const auto kBackgroundColorRoleName = QByteArrayLiteral("backgroundColor");
 } // namespace
 
 struct TokenData {
     QString symbol, name, communityId, communityName, communityImage, collectionUid, collectionName, image;
+    QColor backgroundColor{Qt::transparent};
```

Comment:
I can add it in the next integration PR (already WIP), this one is really just UI (QML) work mostly, didn't want to introduce any model changes

Reply:
Ok! It's just a reminder to introduce it earlier since I'd say that after, it will be more complicated!! But yeah, it's perfect if you do it in the next iteration!